deebot_client/events/mop_auto_wash_frequency.py

```diff
@@ -0,0 +1,24 @@
+"""Mop Auto-Wash Frequency event module."""
+
+from __future__ import annotations
+
+from dataclasses import dataclass
+from enum import IntEnum, unique
+
+from .base import Event
+
+
+@unique
+class MopAutoWashFrequency(IntEnum):
+    """Enum class for all possible mop auto-wash frequencies."""
+
+    TEN_MINUTES = 10
```

Comment:
As the values are numbers, can we also set the interval to 1 minute? And would the bot respect this set one minute?

Reply:
It doesn't seem like it, there may be some other magic values but these are the available presets for the X5 Pro Omni.

Reply:
In the App from T30 I can only chose between 10, 15 and 25 minutes for time based mop auto wash.
The other option is mop after room